server/routerlicious/pnpm-lock.yaml

```diff
@@ -3200,6 +3200,10 @@ packages:
     resolution: {integrity: sha512-aAC7firDZhUhXKU/PjD9Ty7BZt6aTKqOdhsueet7CLgSr/iN+uIsmDCBVIoFsS72bQyZtQYu/61XKegA63+qMg==}
     dev: false
 
+  /@fluidframework/protocol-definitions/3.2.0-236943:
```

Comment:
There is a duplicate entry for protocol-definitions 3.2.0-* now. Probably we should only have one. Can the other references advance as well?